src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -504,10 +504,7 @@ void T() { }
 }
 """;
         var comp = CreateCompilation(src);
-        comp.VerifyEmitDiagnostics(
-            // (5,14): error CS0102: The type 'Extensions.extension<T>(object)' already contains a definition for 'T'
-            //         void T() { }
-            Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "T").WithArguments("Extensions.extension<T>(object)", "T").WithLocation(5, 14));
+        comp.VerifyEmitDiagnostics();
```

Comment:
Consider adding a similar test for an accessor: `extension<get_P>(...) int P => 0;`